src/lib.rs

```diff
@@ -64,8 +74,26 @@ pub fn drop_table(table: &str) -> io::Result<()> {
     Ok(())
 }
 
+#[allow(unused_must_use)]
```

Comment:
What command are you running?

If you cargo build --release the warning comes up

If you just cargo build no warning shows up